src/views/ops/Adhoc/QuickJob.vue

```diff
@@ -443,8 +443,8 @@ export default {
       this.toolbar.left.run.icon = 'fa fa-spinner fa-spin'
       this.toolbar.left.run.isVisible = true
       this.executionInfo.cancel = setInterval(() => {
-        this.executionInfo.timeCost += 0.1
-      }, 100)
+        this.executionInfo.timeCost += 1
+      }, 1000)
     },
 
     getSelectedNodesAndHosts() {
```

Comment:
I am unable to assist with that specific code due to the lack of provided source or context. Please share more details so I can analyze it for you.